### PR TITLE
[JetNews] Bump to alpha07

### PR DESCRIPTION
**Pull request by yrezgui**
I'm currently running into a testing issue both in locally and in GitHub actions

### CODE REVIEW

**Comment by yrezgui** [comment]
I'll merge #266 once this PR is merged

File: JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardTop.kt [hunk 1]
```diff
@@ -54,8 +55,7 @@ fun PostCardTop(post: Post, modifier: Modifier = Modifier) {
         }
         Spacer(Modifier.preferredHeight(16.dp))
 
-        val emphasisLevels = AmbientEmphasisLevels.current
-        ProvideEmphasis(emphasisLevels.high) {
+        Providers(AmbientContentAlpha provides ContentAlpha.high) {
```
Comment (florina-muntenescu) [inline JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardTop.kt:58]: the default is high, so this shouldn't be needed.

File: JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardYourNetwork.kt [hunk 2]
```diff
@@ -62,8 +63,7 @@ fun PostCardPopular(
                     .fillMaxWidth()
             )
             Column(modifier = Modifier.padding(16.dp)) {
-                val emphasisLevels = AmbientEmphasisLevels.current
-                ProvideEmphasis(emphasisLevels.high) {
+                Providers(AmbientContentAlpha provides ContentAlpha.high) {
```
Comment (florina-muntenescu) [inline JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardYourNetwork.kt:66]: the default is high, so this shouldn't be needed. same below

**Review by florina-muntenescu — CHANGES_REQUESTED** [review]
Remove all the high emphasis, as that's the default.

File: JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt [hunk 3]
```diff
@@ -117,19 +116,17 @@ fun PostCardHistory(post: Post, navigateTo: (Screen) -> Unit) {
             modifier = Modifier.padding(end = 16.dp)
         )
         Column(Modifier.weight(1f)) {
-            ProvideEmphasis(AmbientEmphasisLevels.current.medium) {
```
Comment (florina-muntenescu) [inline JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt:120]: this medium emphasis was removed on purpose?

**Review by florina-muntenescu — APPROVED** [review]
one comment but otherwise looks good

File: JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt (hunk 3 above)
Comment (yrezgui) [reply JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt:120]: I should create a bug for that. The migration tool on `AmbientEmphasisLevels` set the value automatically to `ContentAlpha.high` and I removed it after cleaning up the project

**Pull request merged**